Header header
sensor_msgs/Imu imu
sensor_msgs/MagneticField mag_field
string IMU_String
